<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="houseEdge">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>House Edge Calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameStrategies">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameSim">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaming Strategies</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
